FunctionalTableData/CellStyle.swift

```diff
@@ -81,16 +85,59 @@ public struct CellStyle {
 		self.accessoryType = accessoryType
 		self.selectionColor = selectionColor
 		self.backgroundColor = backgroundColor
-		self.backgroundView = backgroundView
+		self.tintColor = tintColor
+		self.layoutMargins = layoutMargins
+		self.cornerRadius = cornerRadius
+
+		struct DefaultBackgroundProvider: BackgroundViewProvider {
```

Comment:
The `DefaultBackgroundProvider` in the deprecated init simply wraps and returns the (optionally) provided `backgroundView`. So we'll still get the functionality in this case.